sdk/lib/cmlib/hivecell.c

```diff
@@ -5,10 +5,22 @@
  *            Copyright 2001 - 2005 Eric Kohl
  */
 
+/* INCLUDES *****************************************************************/
+
 #include "cmlib.h"
```

Comment:
Which purpose does this added comment have? Is the additional "s" after "include" followed by 66 stars really adding any value? I mean: Look what the first word at line 10 is telling you what is comming next... Surprise: The includes.

Reply:
I have the feeling that you clearly have no idea what you're actually reviewing which, granted some reviews may make sense but most of them are just unnecessary noise. Here's a following list of links:

https://github.com/reactos/reactos/blob/master/ntoskrnl/ob/obsecure.c
https://github.com/reactos/reactos/blob/master/boot/freeldr/freeldr/freeldr.c
https://github.com/reactos/reactos/blob/master/base/system/winlogon/winlogon.c
https://github.com/reactos/reactos/blob/master/base/services/shsvcs/shsvcs.c
https://github.com/reactos/reactos/blob/master/drivers/filesystems/ntfs/btree.c
(and so on)

And we have been following such style for ages and nobody has questioned it. I am going to say it for the last time, think twice before reviewing.

Reply:
https://reactos.org/wiki/Coding_Style dosn't mention or enforce anything like that. Other places having chosen questionable practices is not requiring you or should encourage you to do the same.
Nevertheless most other code that you pointed to doesn't suffer as much from commentitis as it is the case for the code that is under review here.
People nitpicking mostly about comments is a logical consequence when >50% of the proposed code-changes do consist of comments just.
But let's not fight, instead please take away something for the future from this discussion.